src/items.js

```diff
@@ -66,6 +68,31 @@ const createHeartElement = (item) => {
   return heartElement;
 };
 
+const addComment = (event) => {
+  event.preventDefault();
+  const { parentElement } = event.target;
+  const nameInput = parentElement.querySelector('input');
+  const commentInput = parentElement.querySelector('textarea');
+  const name = nameInput.value;
+  const comment = commentInput.value;
+  const id = event.target.parentElement.dataset.itemId;
+  items = getItems();
+  items = items.map((item) => {
+    if (item.id === id) {
+      const newComment = {
+        id: uuidv4(),
```

Comment:
`uuid` is a bit of an overkill for this id. Just get the item index and use that. Like so: `items.map((item, i))`

Reply:
Also, take cue from the Like function to edit the items array in-place. A simple `forEach` is enough.